src/main/java/org/gridsuite/modification/server/entities/equipment/modification/VoltageLevelTopologyModificationEntity.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.gridsuite.modification.server.entities.equipment.modification;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import org.gridsuite.modification.dto.ModificationInfos;
+import org.gridsuite.modification.dto.VoltageLevelTopologyModificationInfos;
+import org.gridsuite.modification.server.entities.equipment.modification.attribute.BooleanEquipmentAttributeModificationEntity;
+import org.gridsuite.modification.server.entities.equipment.modification.attribute.EquipmentAttributeModificationEntity;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+/**
+ * @author REHILI Ghazwa <ghazwarhili@gmail.com>
+ */
+
+@NoArgsConstructor
+@Getter
+@Entity
+@Table(name = "voltageLevelTopologyModification")
+@PrimaryKeyJoinColumn(foreignKey = @ForeignKey(name = "voltageLevelTopologyModification_id_fk_constraint"))
+public class VoltageLevelTopologyModificationEntity extends EquipmentModificationEntity {
+
+    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.LAZY)
+    @JoinColumn(
+            name = "voltage_level_topology_modification_id",
+            referencedColumnName = "id",
+            foreignKey = @ForeignKey(
+                    name = "voltageLevel_booleanEquipmentAttributeModification_fk"
```

Comment:
```suggestion
                    name = "voltageLevelTopologyModification_booleanEquipmentAttributeModification_fk"
```
even it is so long but to avoid collision in the futur

Reply:
done